lib/api/request.js

```diff
@@ -18,19 +19,21 @@ const {captureException} = require('../utils/sentry/Raven');
  * @param {function(Response)} [onFail] - will be invoked if response not ok
  * @returns {Promise}
  */
-async function request(url, requestObject, onFail) {
+async function request(url, requestObject = {}, onFail) {
+	if (!requestObject.headers) {
+		requestObject.headers = {};
+	}
+
 	if (requestObject.body) {
 		requestObject.body = JSON.stringify(requestObject.body);
 
-		if (!requestObject.headers) {
-			requestObject.headers = {};
-		}
-
 		if (!requestObject.headers['Content-Type']) {
 			requestObject.headers['Content-Type'] = 'application/json';
 		}
 	}
 
+	setUserAgent(requestObject.headers);
```

Comment:
What is the purpose of this change? `setUserAgent` is already called inside `fetch` on line [40](https://github.com/SuitestAutomation/suitest-js-api/pull/505/files#diff-7ca33c3a208a43200fedd5490d451cc48602a0282ddade8bcacedc18f5d50ab7R40)

Reply:
I see. So these changes are not needed at all.